### PR TITLE
set Reply-to address 

### PR DESCRIPTION
**Pull request by tkimnguyen**
so that replying to an email sent by the contact-info form does the expected thing

### CODE REVIEW

**Comment by vangheem** [comment]
tests running: http://jenkins.plone.org/job/pull-request-5.0/502/

Changelog!

File: Products/CMFPlone/browser/contact_info.py [hunk 1]
```diff
@@ -67,12 +67,13 @@ def send_message(self, data):
 
         try:
             # This actually sends out the mail
-            host.send(
+            host.secureSend(
```
Comment (mauritsvanrees) [inline Products/CMFPlone/browser/contact_info.py]: Why the change to `secureSend`? This method  from SecureMailHost was already deprecated in Plone 4.  It is still there, but we want to get rid of it. See issue #965.

Comment (vangheem) [inline Products/CMFPlone/browser/contact_info.py]: because send doesn't allow setting headers. It's unfortunate the API doesn't allow setting headers of the message.

Comment (mauritsvanrees) [inline Products/CMFPlone/browser/contact_info.py]: Ah, right. I guess alternatively we could set `message['Reply-to']` for the same effect.

Comment (davisagli) [inline Products/CMFPlone/browser/contact_info.py]: Here's the helper I'm using to send mail in one project:

```
from email.Header import Header
from email.mime.multipart import MIMEMultipart
from email.mime.text import MIMEText
from Products.CMFCore.utils import getToolByName
import email


def send_mail(portal, subject, message, mfrom=None, mto=None, attachments=None):
    # encode message if necessary
    if isinstance(message, unicode):
        message = message.encode('utf8')

    # add plain text version of html email
    if message.startswith('<html'):
        portal_transforms = getToolByName(portal, 'portal_transforms')
        text = str(portal_transforms.convert('html_to_web_intelligent_plain_text', message))
        msg = MIMEMultipart('alternative')
        msg.attach(MIMEText(text, 'plain', 'utf-8'))
        msg.attach(MIMEText(message, 'html', 'utf-8'))
    else:
        msg = email.message_from_string(message)
        msg.set_charset('utf-8')

    # add attachments (pass in as list of instances of MIMEImage, etc)
    if attachments:
        text = msg
        msg = MIMEMultipart('related')
        msg.attach(text)
        for attachment in attachments:
            msg.attach(attachment)

    # Set Reply-To header
    # (From header will be the portal's From address with the sender's real name)
    realname = None
    if mfrom is not None:
        realname, replyaddr = email.utils.parseaddr(mfrom)
        msg['Reply-To'] = Header(mfrom, 'utf-8')

    if not realname:
        realname = portal.email_from_name
    mfrom = email.utils.formataddr((realname, portal.email_from_address))

    # Send to portal email address if no recipient was specified
    if mto is None:
        mto = mfrom

    # Actually send
    mailhost = portal.MailHost
    mailhost.send(
        msg, subject=subject, mfrom=mfrom, mto=mto,
        immediate=True, charset='utf-8')
```

Comment (mauritsvanrees) [inline Products/CMFPlone/browser/contact_info.py]: I have updated the code to add a Header instead of using secureSend. See 0eac3fef65a82c7ab2171585bf0c53d1c4af9748
